doc/zh-cn/how-to-build.md

```diff
@@ -34,20 +34,20 @@ Apache APISIX 的运行环境需要 Nginx 和 etcd，
 你需要先下载 Apache Release 源码包：
 
 ```shell
-wget http://www.apache.org/dist/apisix/1.4.1/apache-apisix-1.4.1-src.tar.gz
-tar zxvf apache-apisix-1.4.1-src.tar.gz
+wget http://www.apache.org/dist/apisix/1.5/apache-apisix-1.5-src.tar.gz
+tar zxvf apache-apisix-1.5-src.tar.gz
 ```
 
 安装运行时依赖的 Lua 库：
 ```
-cd apache-apisix-1.4.1
+cd apache-apisix-1.5
 make deps
 ```
 
 ### 通过 RPM 包安装（CentOS 7）
 
 ```shell
-sudo yum install -y https://github.com/apache/incubator-apisix/releases/download/1.4.1/apisix-1.4.1-0.el7.noarch.rpm
+sudo yum install -y https://github.com/apache/incubator-apisix/releases/download/1.5/apisix-1.5-0.el7.noarch.rpm
```

Comment:
update from `https://github.com/apache/incubator-apisix` to `https://github.com/apache/apisix`

Reply:
please fix the other points